src/main/java/org/folio/service/jobnumber/JobNumberService.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.service.jobnumber;
+
+import java.util.Objects;
+
+import io.vertx.core.Future;
+import org.folio.dao.jobnumber.JobNumberDAO;
+import org.folio.rest.core.model.RequestContext;
+import org.folio.rest.jaxrs.model.SequenceNumber;
+import org.folio.rest.persist.DBClient;
+
+public class JobNumberService {
+
+  private final JobNumberDAO jobNumberDAO;
+
+  public JobNumberService(JobNumberDAO jobNumberDAO) {
+    this.jobNumberDAO = jobNumberDAO;
+  }
+
+  public Future<SequenceNumber> getNextNumber(String type, RequestContext requestContext) {
+    if (!Objects.equals(type, "Logs")) {
```

Comment:
We can introduce new enum and check if the value is valid enum value, it will simplify adding new job types in future

Reply:
agree, introduced new enum